### PR TITLE
feat(kms): refacto delete modal

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-15233

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-15233 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/key-management-service/src/components/layout-helpers/Dashboard/GeneralInformationsTiles/BillingInformationsTile.tsx [hunk 1]
```diff
@@ -90,7 +65,8 @@ const BillingInformationsTile = ({
           actionType: 'navigation',
           actions: ['delete_kms'],
         });
-        setShowTerminationModal(true);
+        navigate(ROUTES_URLS.terminateOkms);
+        // setShowTerminationModal(true);
```
Comment (rjamet-ovh) [inline packages/manager/apps/key-management-service/src/components/layout-helpers/Dashboard/GeneralInformationsTiles/BillingInformationsTile.tsx:69]:
```suggestion
```

**Review by pauldkn — DISMISSED** [review]

**Review by rjamet-ovh — DISMISSED** [review]

File: packages/manager/apps/key-management-service/src/components/layout-helpers/Dashboard/GeneralInformationsTiles/BillingInformationsTile.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/key-management-service/src/components/layout-helpers/Dashboard/GeneralInformationsTiles/BillingInformationsTile.tsx]: done fixed
Thanks

Comment (chipp972) [reply packages/manager/apps/key-management-service/src/components/layout-helpers/Dashboard/GeneralInformationsTiles/BillingInformationsTile.tsx]: done fixed
Thanks

File: packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx [hunk 2]
```diff
@@ -20,22 +26,42 @@ export default function TerminateKms() {
...
-      closeModal();
+      navigate('..');
     },
     onError: () => {
       trackPage({
         pageType: PageType.bannerError,
         pageName: 'delete_kms_error',
       });
-      closeModal();
+      navigate('..');
```
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx]: Good idea
I added the other options on the hook :+1:

**Review by ThibaudCrespin — APPROVED** [review]

**Pull request merged**